src/current/v23.2/live-migration-service.md

```diff
@@ -0,0 +1,598 @@
+
+### Flags
+
+|         Flag         |                                                                                                    Description                                                                                                    |
+|----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `--orchestrator-url` | The URL for the orchestrator, using the [configured port](#service-type). This flag is required unless the value is exported as an environment variable using `export CLI_ORCHESTRATOR_URL="{orchestrator-URL}"`. |
+| `--tls-ca-cert`      | The path to the TLS CA certificate. This can also be [exported](#configure-orchestrator-and-client-certificates) as an environment variable using `export CLI_TLS_CA_CERT="{path-to-cli-ca-cert}"`.               |
+| `--tls-client-cert`  | The path to the TLS client certificate. This can also be [exported](#configure-orchestrator-and-client-certificates) as an environment variable using `export "{path-to-cli-client-cert}"`.                       |
+| `--tls-client-key`   | The path to the TLS client key. This can also be [exported](#configure-orchestrator-and-client-certificates) as an environment variable using `export "{path-to-cli-client-key}"`.                                |
+
+## Shadowing modes
+
+The LMS can be configured to shadow production traffic from the source database and validate the query results on the target. The exact behavior is configured with the [`shadowMode`](#shadowing) Helm value.
+
+### `none`
+
+<img src="{{ 'images/v23.2/migrations/lms_none.svg' | relative_url }}" alt="MOLT LMS shadowing mode - none" />
+
+`shadowMode: none` disables shadowing.
+
+- The LMS sends application requests to the source of truth only.
+- Results from the source of truth are returned to the application.
+- Writes must be manually replicated from the source database to the target database.
+
+You can use this mode to perform a [consistent cutover](#consistent-cutover), along with another tool such as [change data capture (CDC)]({% link {{ page.version.version }}/cdc-queries.md %}) that replicates writes to the target database. {% comment %}For an example, see [Consistent cutover without shadowing](#consistent-cutover-without-shadowing).{% endcomment %}
+
+### `async`
+
+<img src="{{ 'images/v23.2/migrations/lms_async.svg' | relative_url }}" alt="MOLT LMS shadowing mode - async" />
+
+`shadowMode: async` writes to both databases.
+
+- The LMS sends application requests to the source of truth and target database in asynchronous threads, and waits only for the source of truth to respond.
+- Results from the source of truth are returned to the application.
```

Comment:
can you add a note here: "if an asynchronous request has not yet completed, subsequent asynchronous requests will be dropped"

Reply:
Added. But how/when do async requests start up again? Only when the last async request is completed?

Reply:
yeah, more or less. i kinda think we may want to avoid documenting / suggesting it as a whole, cc @cotedm wdyt?

Reply:
per @cotedm we'll add a callout clarifying that async shadowing mode is for testing.